examples/data-science/pandas/CDC_NDI_Mortality_Syft_Data_Owner.ipynb

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
<p>Q:  why not <code>df.head(5)</code>   (get the first 5 entries from the table)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
We want enough data to do some interesting stats calculations but not too much to break the serde. I can get about 10k through pretty quickly and still see some of the below queries work.